tests/cloner_test.go

```diff
@@ -2235,14 +2235,6 @@ var _ = Describe("all clone tests", func() {
 
 			f.ForceBindPvcIfDvIsWaitForFirstConsumer(dataVolume)
 
-			By("Verify Quota was exceeded in logs")
-			matchString := strings.Trim(fmt.Sprintf("\"namespace\": \"%s\", \"error\": \"pods \\\"cdi-upload-target-dv\\\" is forbidden: exceeded quota: test-quota, requested", targetNs.Name), " ")
```

Comment:
Do we know if the new quota message still matches the `ErrQuotaExceeded` function that we use in the dv controllers?

Reply:
If you look at the file we still verify a condition and an event contain ErrExceededQuota